cime_config/testmods_dirs/allactive/gis20km/user_nl_mali

```diff
@@ -0,0 +1 @@
+config_am_globalstats_enable = .false.
```

Comment:
@stephenprice , was this necessary to get the tests to run?  It seems like it would be nice to have globalStats enabled in the tests unless there is problem doing so.

Reply:
I agree. This is a short-term workaround to get the mali hist files recognized by the testing framework. If the .am files are included in the outputs, the testing framework does the comparison on those rather than the full hist. files. This is some quirk with the current testing infrastructure, which seems to have very limited support for including / accounting for MPAS component outputs. I've been communicating with @jasonb5 about this and this was one of the suggested workarounds to allow MALI hist files to be included in the SMS comparison tests, for now.

Reply:
Ok, thanks, that's fine then.  For these short tests, if the full output files are being compared, it's not that important that the globalStats files exist.